di_website/templates/general/general_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layouts/default.html' %}
+
+{% load static wagtailcore_tags wagtailimages_tags string_utils card_tags %}
+
+{% block hero %}
+    {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+    <section class="section">
+        <div class="row row--narrow">
+            <div class="l-sidebar">
+                
+                {% include 'includes/partials/side_nav.html'%}
+
+                <div class="l-sidebar__body">
+                    <div class="is-typeset wysiwyg">
+                        {% for block in page.body %}
+                            {% include_block block %}
+                        {% endfor %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    {% include './includes/team_member_quote.html'%}
+
+    {% for content in page.content %}
```

Comment:
Right here... I handled this with `body` block for those under the side bar (typeset), and the `sections` block for those in the wide area of the page. i.e `TypesetBodyMixin` & `SectionBodyMixin`